Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -101,7 +84,7 @@ public class PortfolioStore: ObservableObject {
     rpcService: self.rpcService,
     keyringService: self.keyringService,
     assetRatioService: self.assetRatioService,
-    ipfsApi: self.ipfsApi
+    ipfsApi: nil
```

Comment:
intended but i shouldn't remove it since `userAssetsStore` still needs it. added back in  [ec72b62](https://github.com/brave/brave-ios/pull/7174/commits/ec72b62c5c5a69b0c5b3c85a7c477addd4f6d5c4)